src/test/java/net/sf/jabref/exporter/layout/LatexFieldFormatterTests.java

```diff
@@ -41,6 +56,20 @@ public void preserveNewlineInAbstractField() {
     }
 
     @Test
+    public void preserveMultipleNewlinesInAbstractField() {
+        String fieldName = "abstract";
+        String text = "lorem ipsum lorem ipsum\n\nlorem ipsum lorem ipsum\n";
+        // The newlines are normalized according to the globally configured newline setting in the formatter
+        // Therefore, "\n" has to be replaced by that
+        text = text.replaceAll("\n", Globals.NEWLINE);
```

Comment:
Can you please create the expected text without the replaceAll action? We want to have fast tests, and replaceAll compiles a regular expression internally when used.